### PR TITLE
Upgrade Azure.ResourceManager.PolicyInsights to `package-2022-09` version

### PR DESCRIPTION
**Pull request by samarthlakhotia**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by archerzz — CHANGES_REQUESTED** [review]
@samarthlakhotia You forgot to update the API signature. Run `eng\scripts\Export-API.ps1 policyinsights` to update it.

**Comment by samarthlakhotia** [comment]
@archerzz I have updated the API signature. Let me know if you any other feedback.

**Comment by samarthlakhotia** [comment]
@archerzz Our team thinks it would be important to also release this new addition to track1. So can we hold on the PR until I can get an exemption?

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ azure-arm: true
 csharp: true
 library-name: PolicyInsights
 namespace: Azure.ResourceManager.PolicyInsights
-require: https://github.com/Azure/azure-rest-api-specs/blob/aa8a23b8f92477d0fdce7af6ccffee1c604b3c56/specification/policyinsights/resource-manager/readme.md
-tag: package-2022-03
+require: https://github.com/Azure/azure-rest-api-specs/blob/05a9cdab363b8ec824094ee73950c04594325172/specification/policyinsights/resource-manager/readme.md
```
Comment (ArthurMa1978) [inline sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md:11]: Please update the CHANGELOG file to specify the release day for this new package

**Review by archerzz — APPROVED** [review]

File: sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Generated/Models/Attestation.cs [hunk 2]
```diff
@@ -64,9 +64,13 @@ public Attestation()
...
+        public Attestation(string policyAssignmentId, string id = default(string), string name = default(string), string type = default(string), string policyDefinitionReferenceId = default(string), string complianceState = default(string), System.DateTime? expiresOn = default(System.DateTime?), string owner = default(string), string comments = default(string), IList<AttestationEvidence> evidence = default(IList<AttestationEvidence>), string provisioningState = default(string), System.DateTime? lastComplianceStateChangeAt = default(System.DateTime?), System.DateTime? assessmentDate = default(System.DateTime?), object metadata = default(object), SystemData systemData = default(SystemData))
```
Comment (archerzz) [inline sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Generated/Models/Attestation.cs:73]: There are breaking changes in the interfaces. `assessmentDate` and `metadata` are inserted before `systemData`.
Here are 2 options:
  - Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- Or you need to bump the major version

**Review by archerzz — CHANGES_REQUESTED** [review]
For track1 SDK, make sure you update the following files, according to [the guide](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

- `AzSDK.RP.Props`
- `AssemblyInfo.cs`

**Comment by samarthlakhotia** [comment]
@archerzz @ArthurMa1978 I have updated the PR with the changes requested. Please review and let me know if you have any questions.

**Review by archerzz — APPROVED** [review]

**Comment by archerzz** [comment]
@samarthlakhotia The build failed because version in [csproj](https://github.com/Azure/azure-sdk-for-net/blob/b7a58d2226b6318eac81a39a6e56e8ea1fe98857/sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/Azure.ResourceManager.PolicyInsights.csproj#L3) is inconsistent with the version in `CHANGELOG.md`.

Make sure you run `\eng\common\scripts\Prepare-Release.ps1 Azure.ResourceManager.PolicyInsights -ServiceDirectory policyinsights  -ReleaseDate {your_release_date}` to make sure it's consistent.

**Comment by samarthlakhotia** [comment]
> @samarthlakhotia The build failed because version in [csproj](https://github.com/Azure/azure-sdk-for-net/blob/b7a58d2226b6318eac81a39a6e56e8ea1fe98857/sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/Azure.ResourceManager.PolicyInsights.csproj#L3) is inconsistent with the version in `CHANGELOG.md`.
> 
> Make sure you run `\eng\common\scripts\Prepare-Release.ps1 Azure.ResourceManager.PolicyInsights -ServiceDirectory policyinsights -ReleaseDate {your_release_date}` to make sure it's consistent.

@archerzz Thank you. I have updated the requested file.

**Pull request merged**